### ISSUE TITLE
Possibility of using the Github Wiki section?

### ISSUE THREAD

**Issue by VaasKahnGrim** [danielga/sourcesdk-minimal#20]
Basically just wondering if it would be possible to have the wiki section of this repo used as a quick refrence for those trying to find specific things from the Source SDK. Given how much stuff is in it this could be quite usefull.

For example if somebody were looking for the LZMA section of the SDK could have a page that contains something along the lines of :
Description: Text here
Location: File path to find this portion of the SDK
Functions(with/without examples maybe):

You could possibly also give access to a select few trust worthy to handle the wiki itself so your not the only one having to put in the time and effort to write it(dunno if thats possible on its own, I've never bothered to try that sort of stuff with a shared repo before)

**Comment by danielga** [comment]
I think I just made it public (to edit).

**Comment by VaasKahnGrim** [comment]
Perfect! Thanks man

**Comment by VaasKahnGrim** [comment]
I started setting up a few pages just a little while ago today. Nothing fully built yet but will slowly add to it when I can. I set the pages up with a description, SDK location for files, a functions list and an examples list.

I included a home page with some information on setting everything up for using the SDK and compiling projects.
I also included a section with external links to the garrysmod_common, scanning, and detouring repos.

I'll have to do some cleaning up of things for the home page on the wiki still tho, but considering everything is working fine with the wiki so far I'mma close this issue to keep things clean
